### DISCUSSION TITLE
Example of connection retry?

### DISCUSSION THREAD

**Discussion by cevr** [urql-graphql/urql#2243] [Q&A] [answered]
Is there an example (or an actual implementation) of an exchange that will attempt to reconnect to a graphql server in the event of a connection refusal for whatever reason?

**Comment by kitten** [comment, marked as answer]
Like, via HTTP with the retry exchange? If so, Yea, that's a thing ✌️ https://formidable.com/open-source/urql/docs/advanced/retry-operations/
